pymeasure/instruments/nf/li5660.py

```diff
@@ -0,0 +1,1665 @@
+        values=[-10.5, 10.5]
+    )
+
+    source6_voltage_offset = Instrument.control(
+        ":SOUR5:VOLT:OFFS?", ":SOUR5:VOLT:OFFS %g",
+        """ A property that allow the user to sets/queries the AUX OUT 2 output voltage. 
+            THe range from -10.5 to +10.5, resolution 0.001 digits, unit V.""",
+        validator=truncated_range,
+        values=[-10.5, 10.5]
+    )
+
+    #################################
+    # Status subsystem, 27 Nov 2020 #
+    #################################
+    status_operation_condition = Instrument.measurement(
+        ":STAT:OPER:COND?",
+        """ Queries the operation condition register (OPCR). """
+    )
+
+    status_operation_enable = Instrument.control(
+        ":STAT:OPER:ENAB?", ":STAT:OPER:ENAB %g",
+        """ Sets/queries the Operation Event Enable register (OPEE).
+            Numeric, range 0 to 65535.
+            Regardless of the value specified, the uppermost bit of the 16-bit binary value is 0.
+            The Operation Condition register contains 0 (all disabled).""",
+        validator=truncated_range,
+        values=[0, 65535]
+    )
+
+    #################################
+    # System subsystem, 27 Nov 2020 #
+    #################################
+    system_key_lock = Instrument.control(
+        ":SYST:KLOC?", ":SYST:KLOC %d",
+        """ Sets/queries the front panel key lock function.
+            On: Enables key lock (disables key operation).
+            Off: Disables key lock (enables key operation).""",
+        validator=strict_discrete_set,
+        values={"On": 1,
+                "Off": 0},
+        map_values=True
+    )
+
+    #################################
+    # Triger subsystem, 27 Nov 2020 #
+    #################################
+    triger_delay = Instrument.control(
+        ":TRIG:DEL?", ":TRIG:DEL %g",
+        """ Sets/queries the trigger delay time. Trigger delay time: Time that elapses between trigger 
+            execution and recording of data or starting of the internal timer. 
+            Numeric, range 0 to 100, unit s, resolution 640ns""",
+        validator=truncated_range,
+        values=[0, 100]
+    )
+
+    trigger_source = Instrument.control(
+        ":TRIG:SOUR?", ":TRIG:SOUR %s",
+        """ Sets/queries the trigger source.
+            Manual: Front panel --> TRIG key
+            External: Rear panel TRIG IN signal
+            Bus: Remote control *TRG or :TRIGger[:IMMediate] command, or the GET message """,
+        validator=strict_discrete_set,
+        values={"Manual": "MAN",
+                "External": "EXT",
+                "Bus": "BUS"},
+        map_values=True
+    )
+
+    ############
+    # Methods  #
+    ############
+    def abort(self):
+        """ A property that allow user to abort recording to the measurement data buffer and puts the trigger system in the idle state. """
+        log.info("Recording abort.")
+
+        self.write(":ABOR")
+
+    def data1_format(self, display="Real"):
```

Comment:
As this is a new instrument in the library, I don't specifically see the point of (a lot of) the wrapper methods (i.e. the methods that pass the argument to one of the previously defined properties); or is this specifically about the default values? (that being said, I'm fine with having them)